cicd/forgeops-tests/config/6.5.0-manifest.txt

```diff
@@ -1,14 +1,7 @@
 [DEFAULT]
-legal_notices = Forgerock_License.txt copyrights.txt third-party-copyrights.txt third-party-licenses
-    Apache-2.0.txt BSD-2-Clause.txt BSD-3-Clause.txt BSD.txt Bouncy_Castle_License.txt
-    CC0-1.0.txt CDDL-1.0.txt CDDL-1.1.txt CDDL_or_GPLv2_with_exceptions.txt CPL-1.0.txt
-    EPL-1.0.txt JSON.txt MIT.txt MPL-2.0.txt
 version = 6.5.0
```

Comment:
do we need this since each section specifies a version?

Reply:
Currently not.  But I think it sets the seen for the whole test i.e. all product tests should be checked against this version.